### ISSUE TITLE
Add minutes. 

### ISSUE THREAD

**Issue by sohate** [BigBerny/magicquit#8]
Add minutes please, not only hours. Thanks!

**Comment by BigBerny** [comment]
Thanks for your feedback. It looked a bit messy when always showing minutes. But now it shows minute if it's less than 1 hour. Is this not enough for your use case?

**Comment by sohate** [comment]
@BigBerny Sorry for misunderstanding. I meant minutes when choosing time for quitting app. For example, I don't need hours, I need 1 minute or 10 minutes of inactive. Or instant quit when closing all windows.

**Comment by BigBerny** [comment]
Ok, got it. But I think what you're looking for is that feature which is not implemented yet: https://github.com/BigBerny/magicquit/issues/5
I think if it will close every application if it's 1 minute in the background, it would be super annoying, no?

**Comment by sohate** [comment]
@BigBerny I don't need close every application, only certain. In your app you can choose apps for quitting. I have couple apps that I use and don't need them to be in dock, right now I'm closing them with my hands, your app can automate that process. So, 1 minute (or even seconds) or instant quit would be great.

**Comment by BigBerny** [comment]
I know what you mean. Don't know if and when I have time to add it though.

**Comment by aayushbhatia06** [comment]
(I know this is a free project that I am very grateful for and I am NOT demanding anything in the slightest but out of pure curiosity) are there any updates on this feature?

**Comment by BigBerny** [comment]
Unfortunately, at the moment I don't have time to work (child and own startup) on it but I hope to implement it someday.
Am 28. Aug. 2024, 13:54 +0200 schrieb Aayush Bhatia ***@***.***>:

**Comment by Hydro8** [comment]
Yes it could be very could to kill app when closing last window of this app.

**Comment by 947116** [comment]
+1 on this 'minute feature' :)

edit: found an app doing exactly that: [https://lawand.io/last-window-quits/](https://lawand.io/last-window-quits/)

**Comment by ahyo1217** [comment]
I also hope this feature will be available. I look forward to the day the developer completes it, and I’m truly grateful for creating this product. Although there are alternative products out there, I still prefer this one for its simple interface.
